src/MergeContext.js

```diff
@@ -222,7 +226,7 @@ class MergeContext {
             }
         }
 
-        this._votingDelay = delay;
+        this._votingDelay = this._approvalResult.delayMs;
```

Comment:
If `this._votingDelay` is always `this._approvalResult.delayMs`, then we should remove the former or transform it into a method.